website/server/libs/chat/group-chat.js

```diff
@@ -0,0 +1,18 @@
+import { model as Chat } from '../../models/chat';
+
+// @TODO: Don't use this method when the group can be saved.
+export async function getGroupChat (group) {
+  const groupChat = await Chat.find({groupId: group._id}).limit(200).sort('-timestamp').exec();
```

Comment:
is the limit 400 for some groups?

Reply:
Not that I know of. We talked about doing it for Group Plans, but after we merged this PR.

Reply:
I think it's already in place.
website/server/models/group.js:537:  const MAX_SUBBED_GROUP_CHAT_COUNT = 400;

Reply:
Ah, okay. I can change that.